tests/test_environment.py

```diff
@@ -9,20 +9,20 @@
 from pythonfinder.environment import possibly_convert_to_windows_style_path
 
 
-@pytest.mark.skipif(os.name != 'nt', reason="Only run on Windows")
+@pytest.mark.skipif(os.name != "nt", reason="Only run on Windows")
 def test_possibly_convert_to_windows_style_path():
     # Create a temporary directory
     with tempfile.TemporaryDirectory() as tmpdirname:
         # Get an input path in the form "\path\to\tempdir"
         drive, tail = os.path.splitdrive(tmpdirname)
-        input_path = tail.replace('/', '\\')
+        input_path = tail.replace("/", "\\")
         assert re.match(r"(\\[^/\\]+)+", input_path)
         revised_path = possibly_convert_to_windows_style_path(input_path)
         assert input_path == revised_path
 
         # Get an input path in the form "/c/path/to/tempdir"
-        input_path = '/' + drive[0].lower() + tail.replace('\\', '/')
+        input_path = "/" + drive[0].lower() + tail.replace("\\", "/")
```

Comment:
Most of these changes are style changes, not dead code ...
Can you please submit the PR again without all the noise?

Reply:
Those changes come from the `black` pre-commit hook configured for this project. I can move those changes to a separate commit if you want, but they should be merged too, since every PR will face those issues.

Reply:
Yes, a separate commit is fine. It allows us to see the actually changes. Black is fine, it just creates a lot of noise.

Reply:
I have split the changes into 2 commits